### DISCUSSION TITLE
Guilded

### DISCUSSION THREAD

**Discussion by DakshG07** [catppuccin/catppuccin#1956] [Port Requests]
**Are you available to help in the creation of this port? 😄 (optional)**
Yes

**Additional context 📷 (optional)**
Preview of repository: [DakshG07/catppuccin-guilded](https://github.com/DakshG07/catppuccin-guilded)
![Preview](https://raw.githubusercontent.com/DakshG07/catppuccin-guilded/main/preview.jpg)

**Comment by DakshG07** [comment]
If anyone could help with the Latte port, it would be much appreciated.

**Comment by WalkQuackBack** [comment]
Guilded has been discontinued as of December 19, 2025. This discussion should be closed.
